factory/usr/lib/core/extra-paths

```diff
@@ -19,4 +19,14 @@ if [ ! -d /run/mnt/data/system-data/var/lib/snapd/kernel/"$kernelName"/"$rev" ];
 /run/mnt/kernel/firmware /usr/lib/firmware none bind,x-initrd.mount 0 0
 /run/mnt/kernel/modules /usr/lib/modules none bind,x-initrd.mount 0 0
 EOF
+    for subdir in firmware modules; do
+        drop_d=/usr/lib/systemd/system/sysroot-usr-lib-"$subdir".mount.d
```

Comment:
I do not think we should expect in the long term that  `/usr` will stay writable in initrd. If we start to play with sysexts then this will be read-only. I am fine with `/etc`. Though it feels like a generator thing.

Reply:
Changed to `/etc`. Re: generator, that's true, but we are already touching `/etc/fstab` here for these mounts and having the definition of the drop-in files elsewhere already caused quite a bit of confusion in the past. In principle this is also a bit of a systemd bug and we should be able to remove the drop-in in the future.